tools/public_holidays/update_countries_meta_holidays.py

```diff
@@ -0,0 +1,132 @@
+"""
+Script to generate public holidays for each country/region defined in countries_meta.txt.
+
+Usage:
+    python3  update_countries_meta_holidays.py            # Uses the current year
+    python3  update_countries_meta_holidays.py --year 2025  # Uses the specified year
+
+This script reads country/region names from countries_meta.txt, and generates separate JSON files containing holidays for each entry.
+Output files are stored in data/public_holidays/{map_name}.json
+"""
+import json
+import pycountry
+import holidays
+import os
+import argparse
+import datetime
+import re
+import sys
+
+custom_name_to_iso = {
+    "Saint Barthelemy": "BL",
+    "United States Virgin Islands": "VI",
+    "Brunei": "BN",
+    "Cape Verde": "CV",
+    "Congo-Brazzaville": "CG",
+    "Congo-Kinshasa": "CD",
+    "Czech": "CZ",
+    "Cote dIvoire": "CI",
+    "East Timor": "TL",
+    "Falkland Islands": "FK",
+    "Macedonia": "MK",
+    "Pitcairn Islands": "PN",
+    "Saint Helena Ascension and Tristan da Cunha": "SH",
+    "Republic of Kosovo": "XK",
+    "Russia": "RU",
+    "Palestine": "PS",
+    "Swaziland": "SZ",
+    "The Bahamas": "BS",
+    "The Gambia": "GM",
+    "Turkey": "TR",
+    "UK": "GB",
+    "New Zealand South": "NZ",
+    "New Zealand North": "NZ",
+    "Abkhazia": "GE",
+    "South Ossetia": "GE",
+    "Jerusalem": "IL",
+    "Crimea": "RU",
+    "Campo de Hielo Sur": "AR",
+}
+
+UNWANTED_PHRASES = [
+    "observed",
+    "in lieu",
+    "by old style",
+    "by new style",
+    "non-working",
+    "substituted from",
+    "National",
+    "substitute day"
+]
+
+
+# Setup paths
+script_dir = os.path.dirname(os.path.abspath(__file__))
+project_root = os.path.abspath(os.path.join(script_dir, '../..'))
+input_file = os.path.join(project_root, 'data', 'countries', 'metadata.json')
```

Comment:
```suggestion
input_file = os.path.join(project_root, 'data', 'countries', 'countries.txt')
```
And update it in all places